tests/unit/conftest.py

```diff
@@ -291,3 +300,22 @@ def httpretty_():
 
     httpretty.disable()
     httpretty.reset()
+
+
+@pytest.fixture
+def application_instance(pyramid_request):
+    return factories.ApplicationInstance(
+        consumer_key=pyramid_request.lti_user.oauth_consumer_key,
+    )
+
+
+@pytest.fixture
+def lti_user(pyramid_request):
+    return pyramid_request.lti_user
+
+
+@pytest.fixture
+def oauth_token(lti_user, application_instance):
+    return factories.OAuth2Token(
+        user_id=lti_user.user_id, application_instance=application_instance
+    )
```

Comment:
These three got moved up from `canvas_api/conftest.py` because the `token_store_service()` fixture needs them